stdlib/builtin.rbi

```diff
@@ -64,6 +63,8 @@ module Kernel
   def loop: { () -> void } -> void
   def puts: (*any) -> void
   def eval: (String, ? Integer?, ?String) -> any
+  def Integer: (Numeric | String | _ToI) -> Integer
```

Comment:
It looks like `Integer` accepts any argument. So the following looks better to me.

```
def Integer: (String, Integer) -> Integer
           | (any) -> Integer
```

Reply:
Looks good! Surely, `String` has [`#to_i`](http://ruby-doc.org/core-2.5.3/String.html#method-i-to_i) method. 👍 

But I have a question. `Kernel.#Integer` documentation [says](http://ruby-doc.org/core-2.5.3/Kernel.html#method-i-Integer
):

> Non string values will be converted by first trying to_int, then to_i.

And `Numeric` has [`#to_int`](http://ruby-doc.org/core-2.5.3/Numeric.html#method-i-to_int) method but doesn't have `#to_i` method.

Then, should we have also `_ToInt` interface like the following? 👇 

```
def Integer: (String, Integer) -> Integer
           | (_ToInt | _ToI) -> Integer

interface _ToInt
  def to_int: -> Integer
end
```

Reply:
I pushed 45fcacb and 8e61a5f. Please check it! 🙏